package.json

```diff
@@ -151,24 +150,24 @@
     "@babel/preset-react": "~7.7.0",
     "@babel/runtime": "~7.7.2",
     "@hot-loader/react-dom": "~16.13.0",
-    "@testing-library/dom": "~7.0.4",
+    "@seneca/vorpal": "^2.1.1",
```

Comment:
why the new vorpal dep here?

Reply:
original Vorpal dependency was last published to npm 5 years back and that has many security vulnerabilities including a critical security vulnerability. This forked version of Vorpal addresses those security vulnerabilities. Our long term plan is to completely re-write our cli to use some modern cli library like @boost/cli.

Reply:
👌